.github/workflows/test-models.yml

```diff
@@ -52,7 +52,7 @@ jobs:
         cd $GITHUB_WORKSPACE
         IMPORTER=1 ./setup_venv.sh
         source shark.venv/bin/activate
-        pytest -k 'not benchmark' --ignore=tank/tf/ --ignore=shark/tests/test_shark_importer.py
+        pytest --ignore=tank/tf/ --ignore=shark/tests/test_shark_importer.py
```

Comment:
i would remove it in the perf-linux and perf-macOS bots and leave it disabled in the GH builder